libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.Ask.schema

```diff
@@ -18,6 +18,12 @@
                     "items": {
                         "type": "string"
                     }
+                },
+                "expectedOperation": {
+                    "$role": "expression",
+                    "type": "string",
+                    "title": "Expected operation to assign entity to property",
+                    "description": "Operation to apply to entities by default."
```

Comment:
>description [](start = 21, length = 11)

are there values for this?  enumeration?

Reply:
No--it is open-ended and you can define your own operations.

---
In reply to: [403331816](https://github.com/microsoft/botbuilder-dotnet/pull/3669#discussion_r403331816) [](ancestors = 403331816)